src/rebuild/srv.c

```diff
@@ -2246,10 +2246,19 @@ int
 ds_rebuild_regenerate_task(struct ds_pool *pool, daos_prop_t *prop)
 {
 	struct daos_prop_entry	*entry;
+	char			*env;
 	int			rc = 0;
 
 	rebuild_gst.rg_abort = 0;
 
+	d_agetenv_str(&env, REBUILD_ENV);
+	if (env && !strcasecmp(env, REBUILD_ENV_DISABLED)) {
+		D_DEBUG(DB_TRACE, "Rebuild is disabled\n");
```

Comment:
suggest changing to DB_REBUILD and adding some more context
 D_DEBUG(DB_REBUILD, DF_UUID ": Rebuild is disabled for all pools\n", DP_UUID(svc->ps_pool->sp_uuid)

also in pool_svc_update_map() where the same environment variable is checked, I think that log message could be DB_REBUILD or DB_MD not DB_TRACE.